### PR TITLE
fix: sequelize mode isNewRecord

### PR DESCRIPTION
**Pull request by JimmyDaddy**
rt

### CODE REVIEW

File: lib/bone.js [hunk 1]
```diff
@@ -83,6 +83,14 @@ class Bone {
       rawInitial: {},
       rawUnset: new Set(),
     }));
+
+    Object.defineProperty(this, 'isNewRecord', {
+      value: true,
+      enumerable: false,
+      writable: true,
+      configurable: true,
+    });
```
Comment (cyjake) [inline lib/bone.js:92]: A direct assignment, such as `this.isNewRecord = true`, shall suffice.

Comment (cyjake) [reply lib/bone.js:92]: btw, I did take a look in Active Record of Ruby on Rails. The equivalent property is called `new_record?` (notice the `?` at the end, which indicates it's a boolean). Hence I think `isNewRecord` is perfect for us.

Comment (JimmyDaddy) [reply lib/bone.js:92]: `isNewRecord` should not be  enumerable, a direct assignment will make developer confuse

Comment (cyjake) [reply lib/bone.js:92]: good point.

**Pull request merged**